gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
Or maybe ours were just so intermingled that it couldn't fully de-dup it. Maybe now that we have already simplified once the task of de-duping will be easier.